students/chris_delapena/lesson03/lesson03SlicingLab.py

```diff
@@ -0,0 +1,50 @@
+# Exercise 3.1, Slicing lab
+
+# Exchange first and last items
+def exchange_first_last(seq):
+    return seq[-1:] + seq[1:-1] + seq[:1]
+
+# Remove every other item
+def remove_every_other(seq):
+    return seq[::2]
+
+# Remove first 4 and last 4 items, and then every other items
+def remove_firstFour_lastFour_everyOther(seq):
```

Comment:
All function names (as well as pretty everything in Python except classes) should be snake_case, just like your function names above.